package ConfigAssistor;

message Grade
{
    required uint32 times   = 1;
    required uint32 cash    = 2;
}

message Assistor
{
    required  uint32  level  = 1;
    required  uint32  num    = 2;
    required  uint32  cdTs   = 3;
    repeated  Grade   grades  = 4;
}

message AssistCfg
{
    required Assistor assistInfo = 1; 
}